src/changelog/buildpacks/_posts/2023-06-08-php-default-version-8.1.md

```diff
@@ -0,0 +1,8 @@
+---
+modified_at: 2023-06-08 16:00:00
+title: 'PHP - Default PHP version updated to 8.1.x on scalingo-20 and scalingo-22'
+github: 'https://github.com/Scalingo/php-buildpack'
+---
+
+For a complete view of all available PHP versions, depending on the stack,
+please see [the dedicated page in our documentation](https://doc.scalingo.com/languages/php/start#php-versions).
```

Comment:
We shouldn't hard code doc.scalingo;com when creating link. We should use `post_url`.

```suggestion
please see [the dedicated page in our documentation]({% post_url languages/php/2000-01-01-start %}#php-versions).
```

Reply:
Sure :man_facepalming: